cchecker_web/processing.py

```diff
@@ -60,6 +91,33 @@ def compliance_check(job_id, dataset, checker):
         return False
 
 
+def stdout_output(cs, aggregates, groups, limit, checker):
+    '''
+    Calls output routine to display results in terminal, including scoring.
+    Goes to verbose function if called by user.
+
+    :param CheckSuite cs: Compliance Checker Suite
+    :param dict aggregates: Dictionary of the Compliance Checker results
+    :param list groups: List of results
+    :param int limit: The degree of strictness, 1 being the strictest, and going up from there.
+    :param str checker: The name of the compliance checker test
+    '''
+    points = aggregates['scored_points']
+    out_of = aggregates['possible_points']
+    source_name = aggregates['source_name']
+
+    # Print out the headers, (slightly customized from typical CC report)
+    print('\n')
+    print("-" * 80)
```

Comment:
Could we refactor this at some point?  There are libraries which will abstract away printing of columns to make text tables.

Reply:
yeah this is pretty ugly. I grabbed it (and slightly modified it) from here: https://github.com/ioos/compliance-checker/blob/5bd2afe7380eb5e3c22fed6b314154fb5a9dd07a/compliance_checker/suite.py#L324-L328